docs/buildkit/tutorials.md

```diff
@@ -0,0 +1,151 @@
+# Buildkit tutorials
```

Comment:
IMHO, this is really a tutorial for developing your first core patch. It belongs under "Core Development".

Buildkit is just the package that provides some of the tools. But if you downloaded `git` and `git-scan` and `hub` and manually created build... then you wouldn't have buildkit, but you'd still use similar flow/concepts.